notebooks/text_models/solutions/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>Maybe add the dropout rate as an argument of <code>build_classifier_model</code> and increase it a little before training the model (the training curves I obtained are showing an overfit that could benefit from an increase dropout rate).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>